src/main/java/it/pagopa/pm/gateway/config/ClientConfig.java

```diff
@@ -36,27 +38,46 @@ public class ClientConfig {
 
     private static final String HTTPS_PROXY_HOST_PROPERTY = "https.proxyHost";
     private static final String HTTPS_PROXY_PORT_PROPERTY = "https.proxyPort";
-    @Value("${bancomatPay.client.url}")
-    private String BPAY_CLIENT_URL;
-
-    @Value("${bancomatPay.client.timeout.ms:5000}")
-    private int BPAY_CLIENT_TIMEOUT_MS;
-
-    @Value("${azureAuth.client.maxTotal:100}")
-    private int AZURE_CLIENT_MAX_TOTAL;
-
-    @Value("${azureAuth.client.maxPerRoute:100}")
-    private int AZURE_CLIENT_MAX_PER_ROUTE;
-
-    @Value("${azureAuth.client.timeout.ms:5000}")
-    private int AZURE_CLIENT_TIMEOUT;
 
     @Value("${postepay.client.url}")
     private String POSTEPAY_CLIENT_URL;
 
     @Value("${postepay.client.timeout.ms:5000}")
     private int POSTEPAY_CLIENT_TIMEOUT;
 
+    @Value("${azureAuth.client.config")
+    private String AZUREAUTH_CLIENT_CONFIG;
+
+    @Value("${bancomatPay.client.config}")
+    private String BANCOMAT_CLIENT_CONFIG;
+
+    private final Map<String, String> azureAuthClientConfigValues;
+    private final Map<String, String> bancomatClientConfigValues;
+
+    {
+        azureAuthClientConfigValues = getAzureAuthClientConfigValues();
+        bancomatClientConfigValues = getBancomatClientConfigValues();
```

Comment:
if the intent is to instantiate azureAuthClientConfigValues and bancomatClientConfigValues with the methods below, then you can avoid splitting declaration and instantiation of these maps